fs-person.html

```diff
@@ -216,7 +216,7 @@
 
     /** PORTRAIT **/
     /*
-     * 1. sinotypic names will still use the full name instead of the given-name and family-name
+     * 1. In portrait mode for sinotypic names, we use name parts
```

Comment:
```suggestion
     * 1. In portrait mode for sinotypic names, we use name parts, except for CJK, which still will use the full name
```
Probably should have caught this comment in our last PR to fs-person when we changed it